components/dashboards-web-component/src/designer/components/DashboardRenderer.jsx

```diff
@@ -239,7 +240,7 @@ export default class DashboardRenderer extends Component {
 
     renderGoldenLayout() {
         // This is necessary as this method is called via a ref in render(). Also we cannot pass arguments to here.
-        if (this.goldenLayout) {
+        if (this.goldenLayout || this.unmounted) {
```

Comment:
When navigating from a Dashboard to the Dashboard Listing page this method gets triggered and I only wanted to run this when the dashboard is mounted otherwise this will cause the previously loaded widgets to subscribe back to data-provider endpoint with previous configurations.